cloudformation/elasticsearch.yaml

```diff
@@ -15,16 +15,17 @@ Resources:
         Ref: AWS::StackName
```

Comment:
Do lines 13-15 above need to be reverted also?

Reply:
When I tested it was fine. It is only the Schema variable https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-cognito-userpool.html#cfn-cognito-userpool-schema